### ISSUE TITLE
[processor/interval] Update configuration format

### ISSUE THREAD

**Issue by ArthurSens** [open-telemetry/opentelemetry-collector-contrib#34920]
```
nit:

ignore:
  gauges: true
  summaries: true
feels more elegant, but no strong opinion at all 

             I like @sh0rez 's idea. Though, I would go for `pass_through` instead of `ignore`. To be a bit more explicit
```

_Originally posted by @RichieSams in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34805#discussion_r1736389801_

**Comment by ArthurSens** [comment]
Creating the issue just to make sure we don't forget :)

**Comment by ArthurSens** [comment]
For the record, I also prefer `pass_through`

**Comment by ArthurSens** [comment]
fixed by #34926
